docs/reference/search/profile.asciidoc

```diff
@@ -276,14 +292,22 @@ the equivalent name in Elasticsearch.  The `"description"` field displays the Lu
 is made available to help differentiating between parts of your query (e.g. both `"message:search"` and `"message:test"`
 are TermQuery's and would appear identical otherwise.
 
-The `"time"` field shows that this query took ~15ms for the entire BooleanQuery to execute.  The recorded time is inclusive
+The `"time"` field shows that this query took ~1.8ms for the entire BooleanQuery to execute. The recorded time is inclusive
+of all children.
+
+The `"time_in nanos"` field shows the timing information in an exact, machine readable format in nanoseconds.
 of all children.
 
 The `"breakdown"` field will give detailed stats about how the time was spent, we'll look at
 that in a moment.  Finally, the `"children"` array lists any sub-queries that may be present.  Because we searched for two
 values ("search test"), our BooleanQuery holds two children TermQueries.  They have identical information (type, time,
 breakdown, etc).  Children are allowed to have their own children.
 
+NOTE: The `"time"` field is only intended for human consumption. If you need exact timing values use `"time_in nanos"`. The `"time"`
+field is currently printed by default, but this will change with the next major version where `"time_in_nanos"` will be
+printed by default. The `"time"` field needs to be switched on by using the `?human=false` <<common-options, common options>> flag then.
```

Comment:
I think this last sentence may be confusing as it's about how to migrate to 6.0.0. That should only be in the 6.0.0 migrate guide I think.